src/components/masters/program/ProgramPage.jsx

```diff
@@ -95,11 +95,12 @@ class ProgramPage extends Component {
                     <Hero
                       programTitle={programName}
                       organizationLogo={{
-                        url: 'https://www.edx.org/sites/default/files/school/image/logo/gtx-logo-200x101.png',
-                        alt: 'Georgia Tech Institute of Technology logo',
+                        url: `${process.env.DESIGNER_BASE_URL}${programBranding.organization_logo.url}`,
+                        alt: programBranding.organization_logo.alt,
                       }}
-                      textureImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt-cyber-title_bg_img_440x400.jpg"
-                      coverImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt_cyber_campus_image_1000x400.jpg"
+                      textureImage={`${process.env.DESIGNER_BASE_URL}${programBranding.texture_image}`}
+                      coverImage={`${process.env.DESIGNER_BASE_URL}${programBranding.cover_image}`}
```

Comment:
same as above for `textureImage` and `coverImage`. Both will be absolute URLs. (They will be coming from S3 not the backend)